test/test_dual_constraints.jl

```diff
@@ -0,0 +1,52 @@
+path = joinpath(pwd(), "test_dual_constraints")
```

Comment:
we can fold the tests for results into a single file. No need to have a separate test file for getting the duals.